include/libtorrent/alert_types.hpp

```diff
@@ -56,6 +56,9 @@ POSSIBILITY OF SUCH DAMAGE.
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #endif
 
+// c++/cx keyword
+#undef interface
```

Comment:
i didnt notice that u already renamed it to listen_interface https://github.com/arvidn/libtorrent/commit/013ed36f0913a9d2845a5d55326af4e7cfbdf5ea. ill just remove this fix then